client/src/components/profile-page/EditContent.js

```diff
@@ -17,22 +18,43 @@ export default function EditContent() {
       <div className="edit-photo">
         <MdAddAPhoto />
       </div>
-
-      <FaDog className="fren-icon blur-effect" />
-      <FaBone className="bone-icon blur-effect" />
-      <p className="fren-count blur-effect">4 frens</p>
-      <p className="dog-rating blur-effect">4.8</p>
-      <p className="feedback blur-effect">feedback</p>
-      <div className="feedback-list blur-effect">
-        <li>the goodest boy</li>
-        <li>loves scritches</li>
-        <li>loves walks</li>
+      <div className="top-flex-wrapper">
+        <img className="profile-picture" src={blankProfilePic} alt="doggy" />
+        <img className="profile-bubble blur-effect" src={bubble} alt="#" />
+        <div className="top-icons-and-text blur-effect">
+          <div className="top-flex-frens">
+            <FaDog className="fren-icon" />
+            <p className="fren-count">4 frens</p>
+          </div>
+          <div className="top-flex-rating">
+            <FaBone className="bone-icon" />
+            <p className="dog-rating">4.8</p>
+          </div>
+        </div>
+      </div>
+      <div className="middle-flex-text blur-effect">
+        <h2 className="dog-name">Bud</h2>
+        <p className="profile-bio-text">
+          Hello, I’m Bud and I’m looking for a friend to hang out with, go
+          running with and chase birds. I’m 20kg and love eating toys.
+        </p>
+      </div>
+      <div className="middle-flex-wrapper">
+        <img className="profile-bio" src={bioBubble} alt="#" />
+        <div className="middle-flex-feedback-wrapper ">
+          <p className="feedback blur-effect">feedback</p>
+          <div className="feedback-list blur-effect">
+            <li>the goodest boy</li>
+            <li>loves scritches</li>
+            <li>loves walks</li>
+          </div>
+          <Link to="/profile">
+            <div className="save-button">save</div>
+          </Link>
+        </div>
```

Comment:
This is a lot of markup being added. 😈 
You might want to consider extracting some components here, even if they are exclusive for this page